test/config_test/example_configs_test_setup.sh

```diff
@@ -5,3 +5,5 @@ set -e
 DIR="$TEST_TMPDIR"/test/config_test
 mkdir -p "$DIR"
 tar -xvf "$TEST_SRCDIR"/envoy/configs/example_configs.tar -C "$DIR"
+
+find "$DIR" -type f | grep -c .yaml > "$TEST_TMPDIR"/config-file-count.txt
```

Comment:
would it be safe/possible to check the platform in here - and maybe exclude freebind/freebind.yaml if necessary - so we can remove some of the `if mac/windows...` in the test file ?

Reply:
hmm - maybe it doesnt matter if its just counting the tarred config files - and more explicit having it in the test file.

Im still thinking we should move the decrement to a constant tho - so it is more explicit

Reply:
if im not mistaken - the freebind.yaml file is already excluded from config tar - which basically means we can just remove the decrement altogether - *if its safe to use the tarred file count*